### ISSUE TITLE
/scim2/Me endpoint fails in Console when switching to organizations

### ISSUE THREAD

**Issue by AnuradhaSK** [wso2/product-is#18708]
**Describe the issue:**
- When switching to an organization there is a /t/<tenant>/o/scim2/Me API invocation, which is invalid
- It fails with 404

https://github.com/wso2/product-is/assets/25483865/023683bd-14c0-4fed-80a5-406d6fbe524c

Latest `wso2is-7.0.0-beta4-SNAPSHOT`
 
```
 <!-- Identity Portal Versions -->
        <identity.apps.console.version>2.10.40</identity.apps.console.version>
        <identity.apps.myaccount.version>2.2.68</identity.apps.myaccount.version>
        <identity.apps.core.version>2.0.105</identity.apps.core.version>
        <identity.apps.tests.version>1.6.373</identity.apps.tests.version>
```

**Comment by brionmario** [comment]
We need to only execute the `Scim` call at 1st level orgs and use the `idToken` to get the profile for the sub orgs.

**Comment by AnuradhaSK** [comment]
> We need to only execute the `Scim` call at 1st level orgs and use the `idToken` to get the profile for the sub orgs.

What are the informations we will retrive from scim2/Me endpoint? Can't we get them from ID token even for root level tenants?

If we don't invoke the scim2/Me endpoint for suborgs, what will be the impact for organization's admin users when login to suborganization space of console?

**Comment by savindi7** [comment]
@AnuradhaSK  We used the scim2/Me endpoint to get the username of the super admin. In situations when a username attribute is mapped to the email and the super admin doesn't have an email set, the id token doesn't give a username to display in the UI (refer issue - https://github.com/wso2/product-is/issues/18582)
As a workaround, we used the me endpoint to retrieve the profile info. 

Before this fix, we used the id token and the scim2/Me endpoint wasnt invoked in any org level..

However, we will have to add a fix to the same issue happening to admin users in sub orgs.
